libs/shared/src/consts/handlebar-helpers/handlebarHelpers.ts

```diff
@@ -17,4 +18,5 @@ export const HandlebarHelpers = {
   [HandlebarHelpersEnum.PLURALIZE]: { description: 'pluralize if needed' },
   [HandlebarHelpersEnum.DATEFORMAT]: { description: 'format date' },
   [HandlebarHelpersEnum.UNIQUE]: { description: 'filter unique values in an array' },
+  [HandlebarHelpersEnum.GROUPBY]: { description: 'group by a property' },
```

Comment:
```suggestion
  [HandlebarHelpersEnum.GROUP_BY]: { description: 'group by a property' },
```
As per my suggestion.